charts/axosyslog-collector/values.yaml

```diff
@@ -48,7 +48,14 @@ config:
     opentelemetry: []
 #   E.g.:
 #     - port: 4317
+  parsers:
+    opentelemetry:
+      enabled: false
   destinations:
+    file: []
+#   E.g.:
+#     - path: "/dev/stdout"
+#       template: "$(format-json .*)"
```

Comment:
The file destination has a lot of useful options (this is true for the network destination too).
Can we add `extraOptions` to these 2 destinations so that everyone can inject options not "natively" supported by the chart?

Reply:
Yes, good idea. Added!